app/retail/templates/home/index.html

```diff
@@ -309,16 +298,21 @@ <h3 class="mt-3 mb-2">{{ article.title }}</h3>
   </script>
   <script>
 
-    $('#sustainability').on('click', function(){
-      $(this).find('img').css('display', 'none');
-      $(this).find('iframe').css('visibility', 'visible');
-      $(this).find('iframe').attr('src', 'https://www.youtube.com/embed/i2JKN_6iuiw?autoplay=1');
-    })
+    var options = "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture";
+    var videoTemplate = function(url){
+      return `<iframe
+      class="video-frame"
+      frameborder="0"
+      allow=${options}
+      allowfullscreen="true"
+      src=${url}></iframe>`;
+    };
+
+    $('.video-preview').on('click', function(){
+      var url = $(this).data('url');
```

Comment:
update all `vars` -> `const` / `let`

Reply:
Sure thing :)
Thought to keep it for compatibility, but I see it's widely supported this days.